docs/source/tutorials/code_snippets/mpl_figure_editor.py

```diff
@@ -34,7 +35,7 @@ def _create_canvas(self, parent):
         sizer.Add(mpl_control, 1, wx.LEFT | wx.TOP | wx.GROW)
         toolbar = NavigationToolbar2Wx(mpl_control)
         sizer.Add(toolbar, 0, wx.EXPAND)
-        self.value.canvas.SetMinSize((10,10))
+        self.value.canvas.SetMinSize((10, 10))
```

Comment:
Ha. Looks like `modernize` sneaks in minor style fixes along the way. (Not objecting; but it's good to know.)